src/main/scala/lms/thirdparty/cpu_array_ops/ArrayCPUOps.scala

```diff
@@ -39,6 +41,47 @@ object ArrayCPUTypeLess extends Dsl with ArrayOps {
     }
   }
 
+  def ARRAY_MINUS(a: ARRAY, b: ARRAY, res: ARRAY, size: INT)(implicit _pos: SourceContext) = {
```

Comment:
There is some boilerplate code among `ARRAY_MINUS`, `ARRAY_MULT`, `ARRAY_DIV`, etc., maybe  we need to abstract over them later.